addons/cargo/initSettings.inc.sqf

```diff
@@ -30,6 +30,16 @@ private _category = [ELSTRING(main,Category_Logistics), LSTRING(openMenu)];
     {[QGVAR(paradropTimeCoefficent), _this, true] call EFUNC(common,cbaSettings_settingChanged)}
 ] call CBA_fnc_addSetting;
 
+[
+    QGVAR(unloadOnKilled),
+    "SLIDER",
+    [LSTRING(unloadOnKilled), LSTRING(unloadOnKilled_description)],
+    _category,
+    [0, 100, 1, 0],
```

Comment:
cba can do native percentages
```suggestion
    [0, 1, 0, 1, true], // [_min, _max, _default, _trailingDecimals, _isPercentage]
```

Reply:
I saw that option, but I couldn't find a single other setting using it, including ones in Ace that do 0, 100